hazelcast/src/main/java/com/hazelcast/internal/partition/ChunkSupplier.java

```diff
@@ -34,7 +34,7 @@ public interface ChunkSupplier extends Iterator<Operation> {
     /**
      * @param isEndOfChunk boolean supplier to signal end of chunk.
```

Comment:
Please, update Javadoc(s)

Reply:
done.

Reply:
@ahmetmircik did you push the javadoc update? I still see "@param isEndOfChunk boolean supplier to signal end of chunk."

Reply:
now it should be ok, somehow i missed it.